ims/src/main/java/com/adobe/aio/ims/feign/FeignImsService.java

```diff
@@ -49,24 +52,35 @@ public AccessToken getJwtExchangeAccessToken() {
     return imsApi.getJwtAccessToken(workspace.getApiKey(), context.getClientSecret(), token);
   }
 
+
   @Override
-  public boolean validateAccessToken(String accessToken) {
-    if (!(workspace.getAuthContext() instanceof JwtContext)) {
-      throw new IllegalStateException("AuthContext in workspace not of type `JwtContext`.");
+  public boolean validateAccessToken(String jwtAccessToken) {
+    if (!workspace.isAuthJWT()) {
+      logger.error("AuthContext in workspace not of type `JwtContext`... this only validates JWT Token");
+      return false;
     }
-
-    return imsApi.validateJwtToken(ACCESS_TOKEN, workspace.getApiKey(), accessToken).getValid();
+    return imsApi.validateJwtToken(ACCESS_TOKEN, workspace.getApiKey(), jwtAccessToken).getValid();
```

Comment:
If this method validates only JWT access token, then should we consider renaming this to `validateJWTAccessToken()`?

Reply:
done